Formula/postgresql@15.rb

```diff
@@ -146,6 +150,8 @@ def caveats
     system "#{bin}/initdb", testpath/"test" unless ENV["HOMEBREW_GITHUB_ACTIONS"]
     assert_equal opt_pkgshare.to_s, shell_output("#{bin}/pg_config --sharedir").chomp
     assert_equal opt_lib.to_s, shell_output("#{bin}/pg_config --libdir").chomp
-    assert_equal "#{lib}/postgresql", shell_output("#{bin}/pg_config --pkglibdir").chomp
+    assert_equal "#{opt_lib}/postgresql", shell_output("#{bin}/pg_config --pkglibdir").chomp
+    assert_equal "#{opt_include}/postgresql", shell_output("#{bin}/pg_config --pkgincludedir").chomp
+    assert_equal "#{opt_include}/postgresql/server", shell_output("#{bin}/pg_config --includedir-server").chomp
```

Comment:
I think this looks a little nicer
```suggestion
    assert_equal opt_lib/"postgresql", shell_output("#{bin}/pg_config --pkglibdir").chomp
    assert_equal opt_include/"postgresql", shell_output("#{bin}/pg_config --pkgincludedir").chomp
    assert_equal opt_include/"postgresql/server", shell_output("#{bin}/pg_config --includedir-server").chomp
```